pom.xml

```diff
@@ -354,7 +355,7 @@
 					  <excludeFromFailureFile>exclude-pmd.properties</excludeFromFailureFile>
 				  </configuration>
                 </execution>
-              </executions>              
```

Comment:
I did no edits except adding the \<module>value-object\</module> but github says those spaces are edited. I don't know why